docs/tutorials/create_custom_identity.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Create a Custom Identity
+parent: pages.tutorials
+nav_order: 8
+---
+
+# Create a Custom Identity
+
+{: .no_toc }
+
+## Table of contents
+
+{: .no_toc .text-delta }
+
+1. TOC
+   {:toc}
+
+---
+
+## Quick reference
+
+Out of the box, a FireFly Supernode contains both an `org` and a `node` identity. Your use case might demand more granular notions of identity (ex. customers, clients, etc.). Instead of creating a Supernode for each identity, you can create multiple custom identities within a FireFly Supernode.
+
+## Additional info
+
+- Reference: [Identities](../reference//identities.md)
+- Swagger: <a href="../swagger/swagger.html#/Default%20Namespace/postNewIdentity" data-proofer-ignore>POST /api/v1/identities</a>
+
+### Step 1: Create a signing key for new identity
+
+`POST` `http://localhost:5100`
+
+`Payload`:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": "1",
+  "method": "personal_newAccount",
+  "params": ["myCustomAccountPassword"]
+}
+```
+
+`Response`:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": "1",
+  "result": "0xfa140b38b81b196c8883deacd344367e17598638" // This is the new signing key
+}
+```
+
+### Step 2: Unlock the new signing key on the node
+
+> This must be repeated after every node restart
+
+`POST` `http://localhost:5100`
+
+`Payload`:
+
+```json
```

Comment:
Maybe change this to `js` (rather than `json`) so it doesn't highlight the comment in 🔴